config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```

Comment:
- [ ] Request to the API was blocked due to CORS policy. You need to add `gem 'rack-cors'` to your Gemfile and bundle install it.

  ![kgogog](https://user-images.githubusercontent.com/54848363/196524095-b3a87129-e66f-4ea5-8b91-ad678cc0860d.jpg)

- [ ] Next, you need to uncomment this code and set the origins to asterisk (*) to allow requests from all sites.

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins "*"
  
      resource "*",
        headers: :any,
        methods: [:get, :post, :put, :patch, :delete, :options, :head]
    end
  end
  ```

- [ ] Check the requested changes in the Frontend PR and verify that your frontend project works perfectly and fetches the greeting before resubmitting for another review